lightningd/pay.c

```diff
@@ -1192,6 +1193,7 @@ static struct command_result *json_sendonion(struct command *cmd,
 		   p_opt("shared_secrets", param_secrets_array, &path_secrets),
 		   p_opt_def("partid", param_u64, &partid, 0),
 		   p_opt("bolt11", param_string, &b11str),
+		   p_opt_def("msatoshi", param_msat, &msat, AMOUNT_MSAT(0)),
```

Comment:
New param deserves documenting in `doc/lightning-sendonion.7.md`.

Reply:
Righto, will amend